.env.example

```
@@ -36,6 +36,9 @@ MAX_GROUP_BY_OVERRIDE=3 # Set maximum aloud group bys
 TAG_ENABLED_LIMIT=200 # Set the max amount of tags per account
 DELAYED_TASK_TIME=30 # Set the seconds before a delayed summary task should expire
 DELAYED_TASK_POLLING_MINUTES=5 # Set the time before the delayed task kick off.
+# Seconds of stale Celery parent heartbeat before worker /livez fails (default 60).
+# Long-running child tasks stay live while the parent still ticks.
+PROBE_LIVENESS_HEARTBEAT_SECONDS=60
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PROBE_LIVENESS_HEARTBEAT_SECONDS key should go before the PROCESSING_WAIT_TIMER key

(UnorderedKey)